docs/contributing/code-style/sql.md

```diff
@@ -215,9 +315,271 @@ in Azure SQL.
 CREATE NONCLUSTERED INDEX [IX_OrganizationUser_UserIdOrganizationIdStatus]
    ON [dbo].[OrganizationUser]([UserId] ASC, [OrganizationId] ASC, [Status] ASC)
    INCLUDE ([AccessAll])
-   WITH (ONLINE = ON); -- ** THIS ENSURES ONLINE **
 ```
 
+#### Naming Conventions
+
+- **Indexes**: `IX_{TableName}_{ColumnName(s)}` (e.g., `[IX_User_Email]`)
+  - The name should clearly indicate the table and the columns being indexed
+
+#### Index Best Practices
+
+- Create indexes after table definition with `GO` separator
+- Use descriptive names following `IX_{TableName}_{ColumnName}` pattern
+- Include `INCLUDE` clause when beneficial for covering indexes
+- Use filtered indexes with `WHERE` clause when appropriate
+
+## General Naming Conventions
+
+### Schema and Object Prefixes
+
+- **Schema**: Use `[dbo]` prefix for all objects
+- **Object names**: Always use square brackets `[dbo].[TableName]`
+
+### User Defined Types
+
+- **User Defined Types**: Filenames should be `{TypeName}.sql` (e.g., `GuidIdArray.sql`)
+  - Note: Use sparingly as they cause downstream maintenance and performance problems
```

Comment:
There is a note block you can use, but these are small so I'd just omit saying it at all.